### PR TITLE
Make title a level-one heading instead.

### PR DESCRIPTION
**Pull request by locks**
Partially addresses #119.

cc @melsumner

### CODE REVIEW

File: addon/templates/components/welcome-page.hbs [hunk 1]
```diff
@@ -1,10 +1,10 @@
-<div id="ember-welcome-page-id-selector" data-ember-version="{{emberVersion}}">
+<main id="ember-welcome-page-id-selector" data-ember-version="{{emberVersion}}">
```
Comment (acorncom) [inline addon/templates/components/welcome-page.hbs]: From https://developer.mozilla.org/en-US/docs/Web/HTML/Element/main:

> A document musn't have more than one <main> element that doesn't have the hidden attribute specified.

Should we steer folks toward better HTML defaults by putting this `<main>` tag in our main app blueprint instead? If we put a `main` here we handle accessibility within this addon but don’t resolve the larger initial accessibility setup for new apps ...

File: addon/templates/components/welcome-page.hbs [hunk 2]
```diff
@@ -1,10 +1,10 @@
...
   <div class="columns">
     <div class="tomster">
       <img src="ember-welcome-page/images/construction.png" alt="Under construction">
     </div>
     <div class="welcome">
-      <h2 id="title">Congratulations, you made it!</h2>
+      <h1 id="title">Congratulations, you made it!</h1>
```
Comment (acorncom) [inline addon/templates/components/welcome-page.hbs]: Happy to merge this line 😉

File: addon/templates/components/welcome-page.hbs (hunk 1 above)
Comment (rwjblue) [reply addon/templates/components/welcome-page.hbs]: > Should we steer folks toward better HTML defaults by putting this <main> tag in our main app blueprint instead?

Yes, I think we should address this elsewhere.

Comment (locks) [reply addon/templates/components/welcome-page.hbs]: I would then recommend that we put the main element in `application.hbs`, surrounding the invocation of this component. That way it is visible to the end-programmer and we don't force a main element on the application.

File: addon/templates/components/welcome-page.hbs (hunk 2 above)
Comment (locks) [reply addon/templates/components/welcome-page.hbs]: https://github.com/ember-cli/ember-welcome-page/pull/157

**Pull request closed without merging**